src/Kreta/TaskManager/Application/Organization/CreateOrganizationHandler.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Kreta package.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Kreta\TaskManager\Application\Organization;
+
+use Kreta\SharedKernel\Domain\Model\Identity\Slug;
+use Kreta\TaskManager\Domain\Model\Organization\Organization;
+use Kreta\TaskManager\Domain\Model\Organization\OrganizationId;
+use Kreta\TaskManager\Domain\Model\Organization\OrganizationName;
+use Kreta\TaskManager\Domain\Model\Organization\OrganizationRepository;
+use Kreta\TaskManager\Domain\Model\Organization\OwnerDoesNotExistException;
+use Kreta\TaskManager\Domain\Model\Organization\OwnerId;
+use Kreta\TaskManager\Domain\Model\Organization\OwnerRepository;
+use Kreta\TaskManager\Domain\Model\User\UserId;
+
+class CreateOrganizationHandler
+{
+    private $ownerRepository;
+    private $repository;
+
+    public function __construct(OrganizationRepository $repository, OwnerRepository $ownerRepository)
+    {
+        $this->ownerRepository = $ownerRepository;
+        $this->repository = $repository;
+    }
+
+    public function __invoke(CreateOrganizationCommand $command)
+    {
+        $owner = $this->ownerRepository->ownerOfId(
```

Comment:
We should check for userId, ownerId must be generated. Therefore this will always fail as it will never exist